source/vibe/web/rest.d

```diff
@@ -370,7 +370,14 @@ class RestInterfaceClient(I) : I
 
 			URL url = m_baseURL;
 
-			if (name.length) url ~= Path(name);
+			if (name.length)
+			{
+				string sep = "";
+				if (name[0] != '/' && (!url.pathString.length || url.pathString[0] != '/'))
+					sep = "/";
+				url.pathString = url.pathString ~ sep ~ name;
```

Comment:
Now we cant use `~=` operator here, because it calls the `@property Path path()` method and the constructor and them cause re-encoding of url (`.path` will decoded path, construcor will encode it back), but decoding of URL path is irreversible procedure.
For example if we has a `/foo%2fbar/` path, and we decode it, we will take the `/foo/bar/` path. When we encode it back, we will take the `/foo/bar/` which is not the same thing with `/foo%2fbar/`.